### PR TITLE
Added CrythicYT to the About Us Page + Spelling & Grammar

### PR DESCRIPTION
**Pull request by CrythicYT**
also changed Spelling and Wording of FAQ

### CODE REVIEW

File: About_Us/index.html [hunk 1]
```diff
@@ -190,6 +190,18 @@ <h3 class="profile-name">iOnly</h3>
...
     </div>
   </div>
+  <div class="container">
+    <div class="row">
+      <div class="col-lg-4">
+        <div class="card card-1 border-secondary">
+          <img src="https://cdn.discordapp.com/attachments/908676637741187072/918067390217089034/CrythicYT1.png" alt="" class="avatar rounded-circle">
+          <h3 class="profile-name">CrythicYT</h3>
+          <p style="font-size: smaller;">Hi! Im Crythic, I am a Small Youtuber, A Junior Dev, and a Long time Minecraft Player. My IRL name is Matthew.
+          I work rith Python and HTML and trying to learn others aswell.</p>
```
Comment (Kira272921) [inline About_Us/index.html:200]: `rith` -> `with`

**Review by Kira272921 — CHANGES_REQUESTED** [review]
@CrythicYT Small typo

**Comment by CrythicYT** [comment]
Small Typo Fixed

**Review by Kira272921 — APPROVED** [review]

**Comment by StoneMcYT** [comment]
This still active?

**Comment by tintinkung** [comment]
## 🔏  closed: duplicated from pr #96

**Pull request closed without merging**